GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -377,14 +377,14 @@ private static ArgumentString FindRenamesAndCopiesOpts()
             return FindRenamesOpt() + findCopies;
         }
 
-        public void DisableFilters()
+        public void ResetAllFilters()
         {
-            _filterInfo.DisableFilters();
+            _filterInfo.ResetAllFilters();
         }
 
-        public void DisableRevisionFilter()
+        public void ResetRevisionFilter()
         {
-            _filterInfo.DisableFilters();
+            _filterInfo.ResetAllFilters();
             PerformRefreshRevisions();
         }
```

Comment:
❓ The only difference is `PerformRefreshRevisions()`. Rename to `ResetAllFiltersAndRefresh`?

Reply:
ok